src/libraries/System.Diagnostics.Process/src/GlobalSuppressions.cs

```diff
@@ -0,0 +1,7 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Documentation", "CA1200:Avoid using cref tags with a prefix", Justification = "https://github.com/dotnet/csharplang/issues/320", Scope = "type", Target = "~T:System.Diagnostics.Process")]
+[assembly: SuppressMessage("Documentation", "CA1200:Avoid using cref tags with a prefix", Justification = "https://github.com/dotnet/csharplang/issues/320", Scope = "member", Target = "~T:System.Diagnostics.ProcessStartInfo")]
```

Comment:
I was given the suggestion to use the `O:` prefix for method overload groups and globally suppress rule CA1200 (which prevents using prefixes). This will ensure that when we send the documentation back to MS Docs, we don't lose the method group hyperlinks.

Reply:
Why not suppress it instead in `Directory.Build.props`? That would be cleaner. By doing this, we would need to include a similar file change in all assemblies when we port the documentation.

Reply:
I didn't know. Thanks for the suggestion.

Reply:
Had a conversation with Santi, we will try to disable this globally in `eng/CodeAnalysis.ruleset`. If that doesn't work, we will use `<NoWarn>` in the global `Directory.Build.props`.

Reply:
Ruleset files are deprecated, BTW. @mavasani - should the dotnet/runtime repo try to replace the https://github.com/dotnet/runtime/blob/master/eng/CodeAnalysis.ruleset file? Although I see that dotnet/roslyn has a small .ruleset file too: https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/CodeAnalysisRules.ruleset.

Reply:
@gewarren We actively use `CodeAnalysis.ruleset` in the runtime repo. If we should move to a different method, then we should probably track that in a separate issue. I am not sure if we have a specific need for a ruleset file. Thanks for bringing it up, though.

I opened a PR to move CA1200 from Warning to Info: https://github.com/dotnet/runtime/pull/48612